test/test_ops.py

```diff
@@ -1102,296 +1101,374 @@ def test_bbox_convert_jit(self):
-        output = ops.distance_box_iou_loss(box1, box2, reduction=reduction)
-        # TODO: When passing the dtype, the torch.half fails as usual.
-        expected_output = torch.tensor(expected_output, device=device)
-        tol = 1e-5 if dtype != torch.half else 1e-3
-        torch.testing.assert_close(output, expected_output, rtol=tol, atol=tol)
+        assert_iou_loss(ops.complete_box_iou_loss, box1, box1, 0.0, device=device)
+        assert_iou_loss(ops.complete_box_iou_loss, box1, box2, 0.8125, device=device)
+        assert_iou_loss(ops.complete_box_iou_loss, box1, box3, 1.1923, device=device)
+        assert_iou_loss(ops.complete_box_iou_loss, box1, box4, 1.2500, device=device)
+        assert_iou_loss(ops.complete_box_iou_loss, box1s, box2s, 1.2250, device=device, reduction="mean")
+        assert_iou_loss(ops.complete_box_iou_loss, box1s, box2s, 2.4500, device=device, reduction="sum")
 
-    assert_distance_iou_loss(box1, box1, 0.0)
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.half])
+    def test_empty_inputs(self, dtype, device):
+        assert_empty_loss(ops.complete_box_iou_loss, dtype, device)
 
-    assert_distance_iou_loss(box1, box2, 0.8125)
 
-    assert_distance_iou_loss(box1, box3, 1.1923)
+class TestDistanceBoxIouLoss:
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.half])
+    def test_distance_iou_loss(self, dtype, device):
+        box1, box2, box3, box4, box1s, box2s = get_boxes(dtype, device)
 
-    assert_distance_iou_loss(box1, box4, 1.2500)
+        assert_iou_loss(ops.distance_box_iou_loss, box1, box1, 0.0, device=device)
+        assert_iou_loss(ops.distance_box_iou_loss, box1, box2, 0.8125, device=device)
+        assert_iou_loss(ops.distance_box_iou_loss, box1, box3, 1.1923, device=device)
+        assert_iou_loss(ops.distance_box_iou_loss, box1, box4, 1.2500, device=device)
+        assert_iou_loss(ops.distance_box_iou_loss, box1s, box2s, 1.2250, device=device, reduction="mean")
+        assert_iou_loss(ops.distance_box_iou_loss, box1s, box2s, 2.4500, device=device, reduction="sum")
 
-    assert_distance_iou_loss(box1s, box2s, 1.2250, reduction="mean")
-    assert_distance_iou_loss(box1s, box2s, 2.4500, reduction="sum")
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.half])
+    def test_empty_distance_iou_inputs(self, dtype, device):
+        assert_empty_loss(ops.distance_box_iou_loss, dtype, device)
 
 
-@pytest.mark.parametrize("device", cpu_and_gpu())
-@pytest.mark.parametrize("dtype", [torch.float32, torch.half])
-def test_empty_distance_iou_inputs(dtype, device) -> None:
-    box1 = torch.randn([0, 4], dtype=dtype, device=device).requires_grad_()
-    box2 = torch.randn([0, 4], dtype=dtype, device=device).requires_grad_()
+class TestFocalLoss:
```

Comment:
`<rant>`I took the previous code vs new and diffed them. This indicated that you made only minor modifications to the class which Github fails to highlight. This is where splitting the PR into multiple comes handy and can speed up reviews.`</rant>`

Reply:
Seems I should learn to use ghstack https://github.com/ezyang/ghstack This would make it simple.